initrd/bin/kexec-select-boot

```diff
@@ -1,8 +1,82 @@
 #!/bin/sh
 # Generic configurable boot script via kexec
-set -e -o pipefail
-. /tmp/config
-. /etc/functions
+
+usage() {
+	cat <<End-of-message
+Usage
+=====
+
+Generic configurable boot script via kexec
+	
+$0 -b /boot [options]
+	
+Options
+-------
+
+### -a
+
+Anything provided to the -a argument is appended to the kernel command line (usually comes from parsing the grub, syslinux, BLS files). Ex.
+
+	-a 'export CONFIG_BOOT_KERNEL_ADD="intel_iommu=on intel_iommu=igfx_off"'
```

Comment:
Same. Since board configs exports variable, those variables content are available.
On the following lines, we expend cmdadd and cmdremove with the content of the variables. If present, content is added, otherwise the string is appended with an empty string, so not expended.

https://github.com/osresearch/heads/blob/0d5b3d75e7b84ca85ff252fe5fe23aea990da435/initrd/bin/kexec-boot#L32-L33

Reply:
@jtmoree-github-com The origin of add and remove is from here.